src/BloomBrowserUI/lib/long-press/jquery.longpress.js

```diff
@@ -112,26 +114,30 @@
     var textAreaCaretPosition;
     var storedOffset;
 
-    var popup=$('<ul class=long-press-popup />');
+    var popup;
 
     $(window).mousewheel(onWheel);
 
     function onKeyDown(e) {
-
+        /* we had to disable thes because ckeditor was seeing them and messing things up. Hopefully in the future it can be reinstated:
         // Arrow key with popup visible
-        if ($('.long-press-popup').length>0 && (e.which==37 || e.which==39)) {
-            if (e.which==37) activePreviousLetter();
+        if ($('.long-press-popup').length > 0 && (e.which == 37 || e.which == 39)) {
+            e.stopPropagation(); //stop ckeditor from seeing this (DOESN"T WORK)
+            e.preventDefault();
+            if (e.which == 37) activePreviousLetter();
```

Comment:
Other stuff in PR suggests you meant to suppress the activatePrevious/Next behavior but they seem to be still present here.